### PR TITLE
View styles

### PR DESCRIPTION
**Pull request by Timowest12**
In this PR I:

- Added the data to the controller and view to display the data in UI
- On main page show all users
- Each user has an individual profile page
- On the profile page, the first three posts from the user are displayed
- Each profile links to a page that displays all posts
- On the posts page, all posts for the specific user are displayed

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,51 @@
-<h1>this is the posts main page</h1>
\ No newline at end of file
+<h2>
+  <%= link_to "Go to all users", "/users", class: "button" %>
+</h2>
+<section class="user-profile">
+<div class="d-flex row justify-content-center mt-5">
```
Comment (SajjadAhmad14) [inline app/views/posts/index.html.erb:5]: Please use a method from the post model in `app/views/posts/index.html.erb` to show the five most recent comments on a post.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,53 @@
...
+
+  <div class="profheader card col-md-9 m-1  p-3 d-flex">
+
+
+    <div class='users_img'>
+      <img class='profimg' src='<%=@user.photo%>'>
+    </div>
+
+    <div class="user-info">
+      <h2>User:<%=@user.name%></h2>
```
Comment (SajjadAhmad14) [inline app/views/users/show.html.erb:16]: Please follow the given [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAABSCGEJWVM24D2POZ23UGCJKYRSAX7Q) for users show page. The bio should be under picture and username and the `see all posts` button should be below three recent posts.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Timo, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (Timowest12) [reply app/views/posts/index.html.erb:5]: Added functionality to display five most recent comments for each post

File: app/views/users/show.html.erb (hunk 2 above)
Comment (Timowest12) [reply app/views/users/show.html.erb:16]: All posts button moved to the bottom and bio positioned according to the wireframe design guidelines

**Review by julie-ify — APPROVED** [review]
Hi @Timowest12,

# Status: Approved ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**